platform/swift/source/FatalIssueReporter.swift

```diff
@@ -0,0 +1,180 @@
+// capture-sdk - bitdrift's client SDK
+// Copyright Bitdrift, Inc. All rights reserved.
+//
+// Use of this source code is governed by a source available license that can be found in the
+// LICENSE file or at:
+// https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt
+
+import Foundation
+
+/// Progress indicator for crash reporter initialization
+enum IssueReporterInitState: Equatable {
+    case notInitialized
+    case initializing
+    case initialized(ReporterInitResolution)
+}
+
+/// Final state of an initialized crash reporter
+enum ReporterInitResolution: Equatable, Error {
+    case sent
+    case withoutPriorCrash
+    case missingConfigFile
+    case malformedConfigFile
+    case processingFailure(String)
+}
+
+typealias IssueReporterInitResult = (IssueReporterInitState, TimeInterval)
+
+func measureTime<T>(operation: () -> T) -> (T, TimeInterval) {
+    let start = DispatchTime.now()
+    let result = operation()
+    let end = DispatchTime.now()
+    let duration = Double(end.uptimeNanoseconds - start.uptimeNanoseconds) / Double(NSEC_PER_SEC)
```

Comment:
shouldn't this be a ms value? since it's reported in a field called `_fatal_issue_reporting_duration_ms`

Reply:
It should, nice catch! The change'll be at the other end where its serialized, since `TimeInterval` is a measurement in (fractional) seconds